modules/nw-sriov-about-qinq.adoc

```diff
@@ -11,7 +11,13 @@ In traditional VLAN setups, frames typically contain a single VLAN tag, such as
 
 QinQ facilitates the creation of nested VLANs by using double VLAN tagging, enabling finer segmentation and isolation of traffic within a network environment. This approach is particularly valuable in service provider networks where you need to deliver VLAN-based services to multiple customers over a common infrastructure, while ensuring separation and isolation of traffic.
 
-image::693_OpenShift_QinQ_SR-IOV_CNI_0624.png[QinQ]
+The following diagram illustrates how {product-title} can leverage SR-IOV and QinQ to achieve advanced network segmentation and isolation for containerized workloads.
+
+The diagram shows how double VLAN tagging (QinQ) works in a worker node with SR-IOV support. The SR-IOV virtual function (VF) located in the pod namespace, `ext0` is configured by the SR-IOV Container Network Interface (CNI) with a VLAN ID and VLAN protocol. This corresponds to the S-tag. Inside the pod, the VLAN CNI creates a subinterface using as master, the interface `ext0`. This subinterface adds an internal VLAN ID using the 802.1Q protocol, which corresponds to the C-tag.
```

Comment:
```suggestion
The diagram shows how double VLAN tagging (QinQ) works in a worker node with SR-IOV support. The SR-IOV virtual function (VF) located in the pod namespace, `ext0` is configured by the SR-IOV Container Network Interface (CNI) with a VLAN ID and VLAN protocol. This corresponds to the S-tag. Inside the pod, the VLAN CNI creates a subinterface using as primary, the interface `ext0`. This subinterface adds an internal VLAN ID using the 802.1Q protocol, which corresponds to the C-tag.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m

Reply:
I think in this case we have to use master https://docs.openshift.com/container-platform/4.15/networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network, SME asked me to update primary to master plus it is master in the configuration YAML file.